packages/conventional-changelog-lmc-github/src/templates/template.hbs

```diff
@@ -0,0 +1,17 @@
+{{> header}}
+
+{{#each commitGroups}}
+
+  {{#if title}}
+### {{title}}
+  {{/if}}
+
+  {{#each commits}}
+{{> commit root=@root}}
+  {{/each}}
+{{/each}}
+{{#unless commitGroups}}
+Miscellaneous changes: code refactoring, adding missing tests, changes to the build process or auxiliary tools and libraries such as documentation generation
```

Comment:
```suggestion
Miscellaneous changes
```

If there are no commits from any group, I think its better not to assume there are any `code refactoring, adding missing tests` etc. .  Just "Miscellaneous changes" sounds ok.